infrastructure/demo/backend.conf

```diff
@@ -0,0 +1,5 @@
+bucket="aaq-demo-terraform-state" 
+key="demo/terraform.tfstate"
```

Comment:
Should this correspond to some value in tf_backend or is it something we define? Should it be generally `<environment>/terraform.tfstate`?

Reply:
Answer: Yes. Suggested in Readme (@yvonnegitau please correct me if I'm wrong!)

Reply:
Yes.